docs/smart-agent-quickstart.md

```diff
@@ -1,132 +1,298 @@
+cat <<EOH > /etc/yum.repos.d/signalfx-agent.repo
+[signalfx-agent]
+name=SignalFx Agent Repository
+baseurl=https://dl.signalfx.com/rpms/signalfx-agent/final
+gpgcheck=1
+gpgkey=https://dl.signalfx.com/yum-rpm.key
+enabled=1
+EOH
+
+yum install -y signalfx-agent
+
 ```
 
-Your SignalFx API Token can be obtained from the Organization->Access Token tab in [SignalFx](https://app.signalfx.com).
 
-More detailed installation steps to install via a config management tool or using a containerized agent can be found [here](../README.md#installation).
+## Step 5. Add a New Monitor 
 
-#### Step 2: Configuration
+You may also want to add and configure various monitors for your environment. 
 
-The default configuration file should be located at `/etc/signalfx/agent.yaml` on Linux
-and `\ProgramData\SignalFxAgent\agent.yaml` on Windows.
-Also, by default, the file containing your SignalFx API token should be located at
-`/etc/signalfx/token` on Linux and `\ProgramData\SignalFxAgent\token` on Windows.
+* To [add a new monitor](#Monitors).
+* For [Common configuration options](https://docs.signalfx.com/en/latest/integrations/agent/monitor-config.html).
+* For [Windows monitor configurations](https://docs.signalfx.com/en/latest/integrations/agent/windows.html).
+* For [Linux monitor configurations](https://docs.signalfx.com/en/latest/integrations/agent/monitor-config.html)
 
-In the referenced example agent.yaml configuration files below, the default
-location for the token file is used.
 
-- [Linux Default Configuration File](https://github.com/signalfx/signalfx-agent/blob/master/packaging/etc/agent.yaml)
+### Configure optional metrics for your monitors. 
 
-- [Windows Default Configuration File](https://github.com/signalfx/signalfx-agent/blob/master/packaging/win/agent.yaml)
+See individual monitor pages for your monitor from the lists in Step 5 or see the [Integrations tab](https://app.signalfx.com/#/integrations) in the SignalFx application for Monitor Services.
 
-##### Configure your endpoints
+## Step 6. Add a New Observer 
 
-By default, the Smart Agent will send data to the `us0` realm.
-If you are not in this realm, you will need to explicitly set the
-`signalFxRealm` option in your config like this:
+See [Observers](#Observers).
 
-```
-signalFxRealm: YOUR_SIGNALFX_REALM
-```
+## Step 7. Explore Dashboards to display and compare data from various sources. 
 
-To determine if you are in a different realm and need to
-explicitly set the endpoints, check your profile page in the SignalFx
-web application.
+See [Dashboards](https://docs.signalfx.com/en/latest/dashboards/index.html).
```

Comment:
I think you can delete this one and keep the built-in dashboards. That should link to the general dashboards info.

Reply:
done.